stubs/Foundation/helpers.stubphp

```diff
@@ -177,9 +325,11 @@ function url($path = null, $parameters = [], $secure = null) {}
 function validator(array $data = [], array $rules = [], array $messages = [], array $customAttributes = []) {}
 
 /**
+ * Get the evaluated view contents for the given view.
+ *
  * @param  string|null  $view
- * @param  \Illuminate\Contracts\Support\Arrayable<array-key, mixed>|array<string, mixed>  $data
+ * @param  \Illuminate\Contracts\Support\Arrayable<string, mixed>|array<string, mixed>  $data
  * @param  array<string, mixed>  $mergeData
- * @return ($view is null ? \Illuminate\Contracts\View\Factory : \Illuminate\Contracts\View\View)
+ * @return ($view is null ? \Illuminate\Contracts\View\Factory : (\Illuminate\View\View|\Illuminate\Contracts\View\View))
```

Comment:
Why do we have `\Illuminate\View\View` in `@return`?

Reply:
Good question. I briefly answered [here](https://github.com/psalm/psalm-plugin-laravel/pull/335/commits/3c6485c1ffa8083059737b3846adc44a3f2c43fc) and [here](https://github.com/psalm/psalm-plugin-laravel/pull/335/commits/2a8c68d2cc15c5e95f03158673f53aa60bc8ad1e) (pls see commit body), but in a nutshell - this helps bypass issues with similar annotations in Laravel and Larastan: there is no consistency on using this contract vs implementation. AFAIK, Laravel team wanted to remove this contract as probably nobody created and use alternative implementations, but for some reason they haven't done it in Laravel 10. As result - Laravel still somewhere uses Contract, somewhere implementation and Psalm emit MoreSpecificReturnType or LessSpecificReturnType :). Ideally it should be fixed in Laravel, but this package supports 2 major Laravel versions and big range of minor versions - so we still need to have this hack in place, at least for a while right after fixes.